core/src/main/kotlin/org/evomaster/core/problem/graphql/Table.kt

```diff
@@ -0,0 +1,8 @@
+package org.evomaster.core.problem.graphql
+
+
+class Table {
```

Comment:
what is this file for? add some documentation, eg a as a /** */ JavaDoc

Reply:
done